beacon-chain/rpc/prysm/v1alpha1/validator/attester.go

```diff
@@ -170,24 +167,23 @@ func (vs *Server) proposeAtt(
 		return nil, status.Errorf(codes.Internal, "Could not get attestation root: %v", err)
 	}
 
-	var singleAtt *ethpb.SingleAttestation
```

Comment:
There is no need for this variable anymore. It was used so that we can broadcast a single attestation but send an Electra attestation through the feed. But we can safely send a single attestation.